psi4/driver/driver_cbs.py

```diff
@@ -617,13 +623,125 @@ def scf_xtpl_helgaker_3(functionname: str, zLO: int, valueLO: float, zMD: int, v
         return value
 
     else:
-        raise ValidationError("scf_xtpl_helgaker_3: datatype is not recognized '%s'." % type(valueLO))
+        raise ValidationError("xtpl_exponential_3: datatype is not recognized '%s'." % type(valueLO))
+
+
+### NAMED FUNCTION ALIASES
+def scf_xtpl_helgaker_2(functionname: str,
+                        zLO: int,
+                        valueLO: float,
+                        zHI: int,
+                        valueHI: float,
+                        verbose: bool = True,
+                        alpha: float = None) -> Union[float, core.Matrix, core.Vector]:
+    r"""Alias for the exponential form of the extrapolation scheme for reference energies with two adjacent
+    zeta-level bases. Used by :py:func:`~psi4.cbs`.
+
+    Parameters
+    ----------
+    functionname
+        Name of the CBS component.
+    zLO
+        Lower zeta level.
+    valueLO
+        Lower value used for extrapolation.
+    zHI
+        Higher zeta level. Should be equal to zLO + 1.
+    valueHI
+        Higher value used for extrapolation.
+    alpha
+        Overrides the default :math:`\alpha = 1.63`
+
+    Returns
+    -------
+    value : Union[float, core.Matrix, core.Vector]
+        Returns :math:`E_{total}^{\infty}`, see below.
 
+    Notes
+    -----
+    The extrapolation is calculated according to [1]_:
+    :math:`E_{total}^X = E_{total}^{\infty} + \beta e^{-\alpha X}, \alpha = 1.63`
 
-#def corl_xtpl_helgaker_2(functionname, valueSCF, zLO, valueLO, zHI, valueHI, verbose=True):
-def corl_xtpl_helgaker_2(functionname: str, zLO: int, valueLO: float, zHI: int, valueHI: float, verbose: bool = True, alpha: float = None):
-    r"""Extrapolation scheme for correlation energies with two adjacent zeta-level bases.
-    Used by :py:func:`~psi4.cbs`.
+    References
+    ----------
+
+    .. [1] Halkier, Helgaker, Jorgensen, Klopper, & Olsen, Chem. Phys. Lett. 302 (1999) 437-446,
+       DOI: 10.1016/S0009-2614(99)00179-7
+
+    """
+
+    if type(valueLO) != type(valueHI):
```

Comment:
You do the same validation in `xtpl_exponential_2` - this is unneeded. This applies for all your aliases,